apps/gpsrec/settings.js

```diff
@@ -1,4 +1,4 @@
 (function(back) {
   // just go right to our app - we need all the memory
   load("gpsrec.app.js");
-})();
+})() // FIXME: Since this is called directly, should it still have a semi-colon?
```

Comment:
@gfwilliams what do you say regarding the code comment above?

Reply:
Good point. Maybe make it `/* ... */` just in case we decide to add this inline? I guess in most cases it won't be an issue as pretokenisation is on anyway

Reply:
I meant the FIXME comment re the semicolon. Was your comment in response to that?

Reply:
I wonder if I should leave the semi-colon in instead of removing it?

Reply:
Ahh. Well, yes, it was - but the content of the comment didn't go in. I'd remove it :)

Reply:
And no, sorry. The semicolon should be removed - it's still executed the same way as everything else I think? Either way having no `;` on the end won't cause problems

Reply:
Ok - thanks!